### DISCUSSION TITLE
Max Length of Temperature Probe Wires

### DISCUSSION THREAD

**Discussion by kmontagne** [tagyoureit/nodejs-poolController#842] [REM] [answered]
I want to add an additional water temperature sensor into my automation. I want to measure the temp coming from my cooler. I am already using nodejs-poolController with my Intellicenter setup. I read the REM Wiki entry regarding adding temp probe. I was looking at using the Pentair probe. My Raspberry Pi is in my house. What is the maximum practical distance I can extend the probe's wires?

**Comment by rstrouse** [comment]
The wire to my solar probe is over 100 feet and it works fine.

**Comment by MaxVonEvil** [comment, marked as answer]
There's some [info here](https://www.loxone.com/enen/kb/1-wire-extension/) 

TLDR/excerpt; The maximum length of the 1-wire bus depends on the wiring topology. For linear topology and bus topology with short taps up to 300m, for bus topology with long taps and star topology up to 100m. If sensors are connected parasitically, the total length is also limited to 100m.

**Reply by rstrouse** [reply]
The 10k sensors are not 1 wire digital.  They are analog resistance sensors on an adc.  I the wire resistance is tied to the reference divider so there is very little chance of variation as long as the reference voltage is the same on the divider.   Any variation would be linear and the calibration would take care of that.   My guess is that at 23awg you could probably make 500ft.
